frontend/src/components/common/category/category.tsx

```diff
@@ -1,24 +1,37 @@
 import { StringCase } from 'common/enums/enums';
 import { FC } from 'common/types/types';
 import { Image } from 'components/common/common';
-import { changeStringCase } from 'helpers/helpers';
+import { changeStringCase, getValidClasses } from 'helpers/helpers';
 
 import { getRandomColor } from './helpers/helpers';
 import styles from './styles.module.scss';
 
 type Props = {
   keyName: string;
   name: string;
+  activeCategory?: string;
```

Comment:
```suggestion
  isActive?: boolean;
```

Reply:
this is active category key_name, not a boolean

Reply:
But it could be boolean if you move `activeCategory === keyName` check on categories-list